cmd/flags.go

```diff
@@ -43,6 +46,8 @@ func initFlags(
 			initClientInfoFlags(cmd, cfg)
 		case config.Tbtc:
 			initTbtcFlags(cmd, cfg)
+		case config.Maintainer:
```

Comment:
```suggestion
		case config.TbtcMaintainer:
```

Reply:
Done in b0285cf27f51896294dff419b3dec8cf416e7b49.